onnxruntime/contrib_ops/cuda/utils/dump_cuda_tensor.cc

```diff
@@ -188,21 +235,33 @@ void DumpGpuTensor(const char* name, const Tensor& tensor) {
   std::cout << tensor.Location().ToString() << std::endl;
 
   size_t num_dims = shape.NumDimensions();
-  if (num_dims >= 3) {
-    int dim0 = static_cast<int>(shape.SizeToDimension(num_dims - 2));
-    int dim1 = static_cast<int>(shape[num_dims - 2]);
-    int dim2 = static_cast<int>(shape[num_dims - 1]);
+  if (num_dims >= 4) {
+    int dim0 = static_cast<int>(shape.SizeToDimension(num_dims - 4));
```

Comment:
`auto dim0`?

Reply:
It's fine to use int here. If the type name is long, I would prefer auto.